packages/frontend/apps/electron/src/main/protocol.ts

```
@@ -160,6 +160,48 @@ export function registerProtocol() {
             delete responseHeaders['Access-Control-Allow-Origin'];
             delete responseHeaders['Access-Control-Allow-Headers'];
           }
+
+          // to allow url embedding, remove "x-frame-options",
+          // if response header contains "content-security-policy", remove "frame-ancestors/frame-src"
+          delete responseHeaders['x-frame-options'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)